.features-gen/tests/bdd-tests/features/login.feature.spec.js

```
@@ -0,0 +1,38 @@
+    await When('I fill username "zachranNas+brno@pejsekAKocicka.cz"', null, { page }); 
+    await And('I fill password "spatneheslo"', null, { page }); 
+    await And('I click sign in button', null, { page }); 
+    await Then('I should see error message "Invalid email or password"', null, { page }); 
+  });
+
+});
+
+// == technical section ==
+
+test.use({
+  $test: [({}, use) => use(test), { scope: 'test', box: true }],
```

Biome (2.3.14)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)